jupyter_client/multikernelmanager.py

```diff
@@ -324,3 +336,10 @@ def new_kernel_id(self, **kwargs):
         :return: string-ized version 4 uuid
         """
         return unicode_type(uuid.uuid4())
+
+if sys.version_info >= (3, 5):
+    from .multikernelmanager_async_patch import start_kernel_async
+        # this looks weird as we are extra indented, but we are
+        # indeed defining a conditional method depending on the version
+        # of Python.
+    MultiKernelManager.start_kernel_async = start_kernel_async
```

Comment:
Should this be 
```python
MultiKernelManager.start_kernel = start_kernel_async
```
Otherwise, it's not clear to me how `start_kernel_async()` would get invoked - unless there's some python magic happening that is beyond my limited experience.